### PR TITLE
fix estimateGas

### PR DESCRIPTION
**Pull request by acolytec3**

### CODE REVIEW

File: packages/client/src/rpc/modules/eth.ts [hunk 1]
```diff
@@ -28,7 +29,7 @@ import type { EthereumClient } from '../../index.js'
 import type { EthProtocol } from '../../net/protocol/index.js'
 import type { FullEthereumService, Service } from '../../service/index.js'
 import type { RpcTx } from '../types.js'
-import type { Block, JsonRpcBlock } from '@ethereumjs/block'
+import type { JsonRpcBlock } from '@ethereumjs/block'
```
Comment (jochem-brouwer) [inline packages/client/src/rpc/modules/eth.ts:32]: How come this is added but nothing is done with it in this PR? I don't see anything in the diff :thinking:

Comment (acolytec3) [reply packages/client/src/rpc/modules/eth.ts:32]: It should have moved block from a type import to a regular import

**Comment by jochem-brouwer** [comment]
I have further cleaned up some default values (the estimateGas tests were failing) such that the right base fee is used on the block.

File: packages/client/src/rpc/modules/eth.ts (hunk 1 above)
Comment (jochem-brouwer) [reply packages/client/src/rpc/modules/eth.ts:32]: Oh right yes this makes sense, thanks

Comment (jochem-brouwer) [reply packages/client/src/rpc/modules/eth.ts:32]: (its actually in the diff :sweat_smile: - time to call it a day I guess :smile: )

**Review by jochem-brouwer — APPROVED** [review]
I will approve here, but before merging could you check my further changes, and also check that we can now still run kurtosis with tx-fuzz? :smile:

**Review by g11tech — COMMENTED** [review]
post merge review lgtm

**Pull request merged**